spec/unit/settings/autosign_setting_spec.rb

```diff
@@ -72,7 +72,7 @@
   end
 
   describe "converting the setting to a resource" do
-    it "converts the file path to a file resource" do
+    it "converts the file path to a file resource", :if => !Puppet::Util::Platform.windows? do
       path = File.expand_path('/path/to/autosign.conf')
       settings.stubs(:value).with('autosign', nil, false).returns(path)
       Puppet::FileSystem.stubs(:exist?).with(path).returns true
```

Comment:
Just waiting on the stub below to be updated ... excluding on Windows with `:if => !Puppet::Util::Platform.windows?` *and* then calling `Puppet.stubs(:features).returns(stub(:root? => true, :microsoft_windows? => false))` seems unnecessary. Should pick one of the options.

Reply:
It needs both stubs due to the mock raising an unexpected invocation if I remove `:microsoft_windows? => false`

Reply:
See https://github.com/puppetlabs/puppet/pull/6941/files#diff-3c72f1d02ae24d362f1fc748398f67e1R79